app/controllers/movies_controller.rb

```diff
@@ -0,0 +1,67 @@
+class MoviesController < ApplicationController
+  def index
+    @category   = params[:category] || 'title'
+    @sort_order = params[:sort_order] || 'asc'
+    @search     = params[:search]
+    @active     = 'movies'
+    if @category || @search
+      @movie = Library.movies_search_sort(@search, @category, @sort_order)
+    else
+      @movie = Library.all
+    end
+  end
```

Comment:
Unless you're using the variables, set them as local variables instead of instance variables.